src/main/java/com/meetup/server/user/application/UserService.java

```diff
@@ -61,8 +63,10 @@ public void updateNickname(Long userId, String nickname) {
     }
 
     @Transactional
-    public void deleteUser(Long userId) {
+    public void withdrawUser(Long userId) {
```

Comment:
P3) UserService에서 User를 탈퇴하는 메소드니까 User 단어는 제거해도 괜찮지 않을까요?

```suggestion
    public void withdraw(Long userId) {
```